app/views/maps/_facility_auto_complete.js.erb

```diff
@@ -0,0 +1,44 @@
+var facilities = <%= raw @vamc_facilities.to_json %>;
+
+var substringMatcher = function(strs) {
+    return function findMatches(q, cb) {
+        var matches, substringRegex;
+
+        // an array that will be populated with substring matches
+        matches = [];
+
+        // regex used to determine if a string contains the substring `q`
+        substrRegex = new RegExp(q, 'i');
+
+        // iterate through the pool of strings and for any string that
+        // contains the substring `q`, add it to the `matches` array
+        $.each(strs, function(i, str) {
+            if (substrRegex.test(str)) {
+                matches.push(str);
+            }
+        });
+
+        cb(matches);
+    };
+};
+var facilityNameArr = []
+$.each(facilities, function(key, value) {
+    facilityNameArr.push(value.OfficialStationName)
```

Comment:
is there a way we can match what @emyl3 is doing with the Common Name in parenthesis?

Reply:
maybe I can push what I have rn and open up a WIP PR so you can see what I'm doing @JoshingYou1

Reply:
https://github.com/agilesix/diffusion-marketplace/pull/202/files